docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,10 +1,220 @@
 [[get-started-docker]]
 == Running the {stack} on Docker
 
-The Elastic Docker registry contains Docker images for all the products in the 
-{stack}. For more information, see:
+The Elastic Docker registry contains Docker images for all the products in 
+the {stack}: https://www.docker.elastic.co/
+
+[float]
+[[run-stack-docker]]
+=== Run with Docker Compose
+
+To get the default distributions of {es} and {kib} up and running in Docker, 
+you can use `docker-compose`:
```

Comment:
> `docker-compose`

Not sure if the literals are needed here. I'd use "Docker Compose" but don't feel strongly.